Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java

```diff
@@ -34,6 +34,7 @@
 	@PostConstruct
 	public void init() {
 		admins = new HashMap<String, String>();
+		admins.put("admin","admin");
```

Comment:
no, don't hard-code username and password! Add the ones from the sprint-security file.